src/components/pick-list/shared-list-tests.ts

```diff
@@ -186,7 +186,7 @@ export function keyboardNavigation(listType: ListType): void {
       await page.keyboard.press("Tab");
       await page.keyboard.press("Tab");
 
-      expect(await getFocusedItemValue(page)).toEqual("one");
```

Comment:
This test is unrelated. Can you revert it and update the corresponding input E2E tests:

* https://github.com/Esri/calcite-components/blob/master/src/components/input-number/input-number.e2e.ts
* https://github.com/Esri/calcite-components/blob/master/src/components/input-text/input-text.e2e.ts
* https://github.com/Esri/calcite-components/blob/master/src/components/input/input.e2e.ts

?

Reply:
Apologies, this is related, but needs to be updated differently. The double tab key presses were meant to move focus from the input, to the clear button and then to the next item. Since this makes the clear button not focusable, that second tab press should no longer be needed (should now tab from input to item). The assertion needs to stay the same.

Reply:
Hello @jcfranco, I have applied your suggestion. ☺️

Reply:
Awesome! Thanks for making the changes. 

I think we'll be able to merge this w/o adopting. 🎉  I scheduled another E2E test run since it looks like it failed on an unrelated test.